File: c++/cpp_hello/105.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    int n_son, count;
    cin >> n_son;
    double list[n_son];

    for (count = 0; count < n_son; count++){
        cin >> list[count];
    
    }
    int a, b;
    cin >> a >> b;  
    double sum_a_b = 0,Summa = 0;
    for (count = 0; count < n_son; count++){
        if (count < a-1) {
            Summa += list[count];
            sum_a_b ++;
            
        }
        else if ( count >= b){
            Summa += list[count];
            sum_a_b ++;
        }   
        
    }
    
    printf("%.2f", (Summa / sum_a_b));
        
}

File: python/alogo_uz/105.py
n_son = int(input())

massiv = list(map(int, input().split()))

a, b = map(int, input().split())

summa = 0
count = 0 
for i in range(n_son):
    print(i)
    if i < a - 1 or i > b - 1: 
        summa += massiv[i]
        count += 1
        print(massiv[i], " ", i)

print(summa / count)